static/app/views/issueDetails/groupReplays/useReplaysFromIssue.spec.tsx

```diff
@@ -59,6 +60,42 @@ describe('useReplaysFromIssue', () => {
     });
   });
 
+  it('should fetch a list of replay ids for a performance issue', async () => {
+    const MOCK_GROUP = TestStubs.Group({IssueCategory: IssueCategory.PERFORMANCE});
```

Comment:
should the field name be lower case?

```suggestion
    const MOCK_GROUP = TestStubs.Group({issueCategory: IssueCategory.PERFORMANCE});
```